File: Exemplos/App_eXperience_Java/E1_AppExperience_Java-font/app/src/main/java/com/elgin/elginexperience/shipay/models/wallet/Wallet.java
package com.elgin.elginexperience.shipay.models.wallet;

import com.google.gson.annotations.SerializedName;

public class Wallet {
    @SerializedName("active")
    public boolean active;
    @SerializedName("form")
    public WalletForm form;
    @SerializedName("minimum_payment")
    public float minimum_payment;
    @SerializedName("name")
    public String name;
    @SerializedName("wallet_fee")
    public float wallet_fee;
}

File: M8/Exemplos/App_eXperience_Kotlin/app/src/main/java/com/example/e1_elgin_kotlin/shipay/models/wallet/Wallet.kt
package com.example.e1_elgin_kotlin.shipay.models.wallet

import com.google.gson.annotations.SerializedName

data class Wallet (

    @SerializedName("active")
    var active: Boolean,

    @SerializedName("form")
    var form: WalletForm,

    @SerializedName("minimum_payment")
    var minimum_payment : Float,

    @SerializedName("name")
    var name: String,

    @SerializedName("wallet_fee")
    var wallet_fee : Float
)